src/v2/Apps/Artist/Components/ArtistCollectionsRail/ArtistCollectionsRail.tsx

```diff
@@ -44,6 +44,8 @@ export class ArtistCollectionsRail extends React.Component<
 
   render() {
     const { collections } = this.props
+    const isMobile = getENV("IS_MOBILE") === true
```

Comment:
Note that ENV vars are always strings, and `getENV` util doesn't coerce values to boolean, number, etc:

```
isMobile = true === "true" -> `false` 
```